tools/fluent/basic_syntax.md

```diff
@@ -113,28 +113,64 @@ emails = { $unreadEmails ->
     }
 ```
 
-## Variants and Terms
+## Terms and parameterized terms
 
 As described at the beginning of the document, **terms** are a special type of **messages**. They are used to define translations of common words and phrases, which can then be used inside of other messages. They can be recognized because of the identifier starting with a dash, e.g. `-brand-short-name`. Terms can also define additional language-specific attributes which are not present in the reference language (typically `en-US`).
 
-While in most cases terms will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key. Variants represent different forms of the same value. They can be used to define grammatical cases or any other language-specific modifications of the value required by the grammar of the spelling rules. When referencing a term from another message, you can specify which variant to use with the `-term-identifier[variant name]` syntax.
+While in most cases terms will have a single value, it’s also possible to define multiple values associated to a parameter. These parameterized values can be used to define grammatical cases or any other language-specific modifications of the value required by the grammar of the spelling rules. When referencing a term from another message, you can specify a *parameter* and its *value* with the `-term-identifier(parameter: "value")` syntax.
```

Comment:
> ... it’s also possible to define multiple values associated to a parameter ...

Perhaps better wording would be: `associated with a parameter` or `controlled by a parameter`?

> When referencing a term from another message, you can specify a *parameter* and its *value* with the `-term-identifier(parameter: "value")` syntax.

`...you can optionally specify...`